master/internal/db/postgres_groups.go

```diff
@@ -77,5 +89,22 @@ func (db *PgDB) GetUsersInGroup(ctx context.Context, gid int) ([]model.User, err
 		Where("ugm.group_id = ?", gid).
 		Scan(ctx)
 
+	if len(users) < 1 || errors.Is(err, sql.ErrNoRows) {
+		return users, ErrNotFound
+	}
```

Comment:
Same as above: I think this search should return no error if we successfully search but the set is empty.